wrapper/src/api/util.rs

```diff
@@ -10,10 +10,12 @@ pub fn to_wei(eth: String) -> U256 {
     }
 }
 
-pub fn to_eth(wei: String) -> U256 {
+pub fn to_eth(wei: String) -> String {
     let wei = match U256::from_dec_str(&wei) {
         Ok(w) => w,
         Err(_) => panic!("Invalid Wei number: {}", wei),
     };
-    format_ether(wei)
+
+    let parsed_eth = format_units(wei, "ether").unwrap();
+    parsed_eth.trim_end_matches("0").trim_end_matches(".").to_string()
```

Comment:
This will incorrectly remove zeros if there is no decimal `.` (ex: `10000` -> `1`).

I think it'd be safer to do something like this:
```
if (parsed_eth.contains("."))
  parsed_eth.trim_end_matches("0").trim_end_matches(".").to_string()
```

Would also like if we added unit tests inside this `utils.rs` file that test the following cases:
```
1000 -> 1000
100.0 -> 100
100.070 -> 100.07
```

Reply:
nice catch - will update accordingly